holy-books.org

```diff
@@ -525,12 +525,23 @@ a special key ‘:name’ whose value is the Arabic-English name of the chapter.
 
 #+begin_src emacs-lisp
 (defvar holy-books-quran-translation "131"
-  "Variable the defines the translation to be used.
+  "
+The translation code of the Quran; a string.
+
+Possible codes include
+
+Code  Translation
+--------------------
+131   Dr. Mustafa Khattab, the Clear Quran (Default)
+20    Sahih International
+85    Abdul Haleem
+19    Picktall
+22    Yusuf Ali
+95    Abul Ala Maududi
+167   Maarif-ul-Quran
+57    Transliteration
 
-        131 -- Dr.  Mustafa Khattab, the Clear Quran
```

Comment:
@ashfaqfarooqui Maybe we want to keep this URL around so that others are aware of it ;-)

Reply:
Sure, I'll do that.

Here is a better link, but for some reason, the code for Dr. Mustafa Khattab does not match up; and 167 is missing. 
https://api.quran.com/api/v3/options/translations

Reply:
Woah; we can definitely automate retrieval of such a list with a bit of `s-split` ;-)